File: Reptile-SpringBoot/src/main/java/cn/edu/swu/zl/reptilespring/entity/Video.java
package cn.edu.swu.zl.reptilespring.entity;

public class Video extends VideoRaw{

    private long likeIncremental;
    private long commentIncremental;
    private long collectIncremental;

    public long getLikeIncremental() {
        return likeIncremental;
    }

    public void setLikeIncremental(long likeIncremental) {
        this.likeIncremental = likeIncremental;
    }

    public long getCommentIncremental() {
        return commentIncremental;
    }

    public void setCommentIncremental(long commentIncremental) {
        this.commentIncremental = commentIncremental;
    }

    public long getCollectIncremental() {
        return collectIncremental;
    }

    public void setCollectIncremental(long collectIncremental) {
        this.collectIncremental = collectIncremental;
    }
}

File: Reptile-Android/app/src/main/java/cn/edu/swu/reptile_android/model/entity/Video.kt
package cn.edu.swu.reptile_android.model.entity

data class Video(
    val id: Int,
    val videoName: String,
    val likeNum: String,
    val commentNum: String,
    val collectNum: String,
    val userName: String,
    val url: String,
    val videoUrl: String,
    val likeIncremental: Int,
    val commentIncremental: Int,
    val collectIncremental: Int
)